SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -799,33 +813,33 @@ bool USpatialSender::SendRPC(const FPendingRPCParams& Params)
 		Worker_ComponentId ComponentId = SchemaComponentTypeToWorkerComponentId(RPCInfo.Type);
 
 		const UObject* UnresolvedObject = nullptr;
-		Worker_CommandRequest CommandRequest = CreateRPCCommandRequest(TargetObject, Params.Payload, ComponentId, RPCInfo.Index, EntityId, UnresolvedObject);
+		Worker_CommandRequest CommandRequest = CreateRPCCommandRequest(TargetObject, Payload, ComponentId, RPCInfo.Index, EntityId, UnresolvedObject);
 
 		if (UnresolvedObject)
 		{
-			return false;
+			return ERPCResult::UnresolvedTargetObject;
```

Comment:
This should never happen, so maybe a `verify`?

Reply:
Added ensure in corresponding functions, removed unused code.